lapce-data/src/data.rs

```diff
@@ -275,6 +320,135 @@ impl LapceData {
         }
     }
 
+    /// Parses a path into file component and LineCol (line + column combined) components
+    pub fn parse_path_line_column(path: &Path) -> Option<(PathBuf, Option<LineCol>)> {
+        // Convert path to str if path is UTF-8 valid
+        if let Some(str) = path.to_str() {
+            // Path components separated by :
+            let mut splits = str.rsplit(':');
+            // First right component
+            if let Some(first_rhs) = splits.next() {
+                // Is the first right component a number?
+                match first_rhs.parse::<usize>() {
+                    // First right component is a number
+                    Ok(first_rhs_num) => {
+                        // Second right component
+                        if let Some(second_rhs) = splits.next() {
+                            // Is the second right component a number?
+                            match second_rhs.parse::<usize>() {
+                                // Second right component is a number
+                                Ok(second_rhs_num) => { 
+                                    // Interpret [all components to the left of rhs2] combined as path
+                                    let path_str:String = splits.rev().map(|s| s.to_string()).collect::<String>();
+                                    let path_buf = PathBuf::from(path_str);
+                                    let path = path_buf.as_path();
+                                    
+                                    // rhs1 must be column, rhs2 must be line, and everything else to the left combined must be file name
+                                    if path.is_file() {
+                                        return Some((
+                                            path.to_path_buf(),
+                                            Some( LineCol { line: second_rhs_num, column: first_rhs_num } ),
```

Comment:
Should be
```rust
LineCol { line: second_rhs_num, column: first_rhs_num }
```

Reply:
Need to clarify the arguments for `LapceUICommand::JumpToLineColLocation` and `EditorLocation`. If with `position = None` and `scroll_offset = None` then it is automatically accepted that `position = 1, scroll_offset = (1, 1)`, then it is better return `None`